app/services/proofing/aamva/request/verification_request.rb

```diff
@@ -62,34 +62,43 @@ def add_user_provided_data_to_body
           user_provided_data_map.each do |xpath, data|
             REXML::XPath.first(document, xpath).add_text(data)
           end
-          add_street_address_line_2_to_rexml_document(document) if applicant.address2.present?
+
+          add_optional_element(
+            'ns2:AddressDeliveryPointText',
```

Comment:
Out of curiosity, is `ns2` for optional attributes, while `ns1` is for required? or...?

Reply:
The AAMVA SOAP request template we use includes [4 namespaces](https://github.com/18F/identity-idp/blob/main/app/services/proofing/aamva/request/templates/verify.xml.erb#L1):

| Abbrev | URL |
| -- | --|
|`xmlns:soap` |http://www.w3.org/2003/05/soap-envelope|
|`xmlns:ns` |http://aamva.org/dldv/wsdl/2.1|
|`xmlns:ns1`|http://aamva.org/niem/extensions/1.0|
|`xmlns:ns2`|http://niem.gov/niem/niem-core/2.0|

we could probably clean up those `ns*` abbreviations to be more like what they are.